packages/@aws-cdk/aws-dynamodb/lib/table-time-to-live-handler/utils.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-console */
+import type { OnEventRequest } from '@aws-cdk/custom-resources/lib/provider-framework/types';
+import { AWSError, DynamoDB } from 'aws-sdk'; // eslint-disable-line import/no-extraneous-dependencies
+const dynamodb = new DynamoDB({ apiVersion: '2012-08-10' });
+
+
+export async function disableTimeToLive(event: OnEventRequest) {
+  console.log('Trying to disable time to live.');
+  try {
+    await dynamodb.updateTimeToLive({
+      TableName: event.ResourceProperties.TableName,
+      TimeToLiveSpecification: {
+        AttributeName: event.OldResourceProperties?.TimeToLiveAttribute,
+        Enabled: false,
+      },
+    }).promise();
+  } catch (err) {
+    // Catch exception so we can try disabling again at a later point in time.
+    // This is done to mimic the necessary behavior in the enable functionality.
+    const awsError = err as AWSError;
+    if (
+      awsError.code === 'ValidationException' &&
+      awsError.message === 'Time to live has been modified multiple times within a fixed interval'
+    ) {
+      console.log('Time to live has been modified multiple times within a fixed interval. Try again later.');
+      return;
+    }
```

Comment:
This is copy-pasted from the `enableTimeToLive` function. We need to extract this to a common function.